src/modules/task-runner.js

```diff
@@ -60,8 +51,8 @@ class TaskRunner {
       return true;
     }
 
-    if (this.shouldCommandRetry(command)) {
-      sleep.usleep(command.retry.delay * 1000000);
+    if(this.shouldCommandRetry(command)) {
+      snuze.snooze(command.retry.delay * 1000);
```

Comment:
Magic number - assumed microseconds - could this be a named variable e.g. snoozeMicroseconds?